README.md

```diff
@@ -1 +1,253 @@
-# Budget-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget-App
+
+
+## Description
+
+> The Ruby on Rails capstone project ([remember what they are?](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/capstone_intro.md)) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+- I created a Ruby on Rails application that allows the user to:
+
+  - register and log in, so that the data is private to them.
+  - introduce new transactions associated with a category.
+  - see the money spent on each category.
+
+### Screenshots 📸
+
+    Categories
+![](./app/assets/images/img1.png) 
+
+
+    Category-Details 
+![](./app/assets/images/img5.png) 
+
+
+
+## Learning objectives
+
+- Use ruby gems as software packages system.
+- Install Ruby on Rails framework.
+- Understand Rails RESTful design and router.
+- Use controllers to handle requests and render empty views.
+- Use params from browser request in a safe way.
+- Use preprocessed html file with embedded Ruby code.
+- Use layouts and templates for shared content.
+- Use database migration files to maintain database schema.
+- Use validations for models.
+- Secure app from n+1 problems.
+- Understand what ORM is.
+- Write SQL queries with ActiveRecord.
+- Set up associations between models.
+- uild a webapp that requires the user to log in.
+- Use devise gem for authentication.
+- Limit access to webapp resources based on authorization rules.
+- Analyze in writing why you have made a coding choice using one structure over another.
+
+## Live Demo 🔗
+
+[Live Demo Link](https://evans-budget-app.herokuapp.com/)
```

Comment:
- [ ] Please take a look at the live demo, as right now it's not really working. After I click on the login or signup button, heroku throws an error saying that `something went wrong`. It may be because any problem while building the app using heroku, probably a bad deployment, or any missing setup. Please take a look.